docs/Zendesk-batchsource.md

```diff
@@ -56,7 +49,19 @@ If no value is provided, no upper bound is applied.
 **Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records that have a Zendesk score
 equal to the specified score. Only applicable for the Satisfaction Rating object.
 
-### Advanced
+### Connection
+
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use. Object Names information will be provided by the connection.
```

Comment:
Should read Object Name, not Names since we just read one object?

Reply:
Done